hbase-server/src/main/java/org/apache/hadoop/hbase/mob/MobUtils.java

```diff
@@ -737,20 +766,20 @@ public static byte[] serializeMobFileRefs(SetMultimap<TableName, String> mobRefS
       StringBuilder sb = new StringBuilder(100 + mobRefSet.size() * 105);
       boolean doubleSlash = false;
       for (TableName tableName : mobRefSet.keySet()) {
+        if (doubleSlash) {
```

Comment:
I'll merge this change in its own HBASE-25970 commit so it will be easier to track in Jira.